main.go

```diff
@@ -87,14 +86,55 @@ func handleUserRequest(responseWriter http.ResponseWriter, request *http.Request
 			panic(err)
 		}
 
-		// TODO create User
-		userId := UserId{Value: 1}
+		userId, err := databaseutil.CreateUser(
+			signupForm.DisplayName,
+			signupForm.EmailAddress,
+			signupForm.Password)
+		if err != nil {
+			panic(err)
+		}
+
+		userIdObject := UserId{Value: userId}
```

Comment:
personally, i prefer to think of the raw value as `userIdValue`. And the object as `userId`. The idea is to pass around the object and never the value

Reply:
and i'm just now realizing that `CreateUser()` should return `UserId`. No need to make this change in this PR. we can discuss and fix later

Reply:
It should return a User, i still not sure i agree with your first comment. In general i think objects having a field name value is bad. The most extensible flexible thing to me would that the user object would have an Id method.

Reply:
kk sounds like we should discuss this over the phone